server/tinylicious/src/services/inMemorydb.ts

```diff
@@ -23,3 +23,16 @@ export class InMemoryDb extends EventEmitter implements IDb {
         return this.collections.get(name);
     }
 }
+
+export class InMemoryDbFactory implements IDbFactory {
+    private readonly db;
+
+    constructor() {
+        this.db = new InMemoryDb();
+    }
+
+    public async connect(): Promise<IDb> {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Same here: why disable this eslint rule? Instead, doesn't

private readonly db: LevelDb;

solve the issue?